packages/cf-component-select/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "react": "^0.14.2 || ^15.0.0-0",
-    "react-select": "1.0.0-rc.2 || 1.x"
+    "react-select": "1.0.0-rc.3 || 1.x"
```

Comment:
I realize it was RC 2 before but shouldn't we be depending on a more stable version?  Does the functionality we need exist in 1.0.0+?